tests/foreman/api/test_syncplan.py

```diff
@@ -534,8 +536,9 @@ def test_positive_remove_product(module_org):
 
     :BZ: 1199150
     """
-    sync_plan = entities.SyncPlan(enabled=False, organization=module_org).create()
-    products = [entities.Product(organization=module_org).create() for _ in range(2)]
+    sync_plan = target_sat.api.SyncPlan(enabled=False, organization=module_org).create()
+    products = [target_sat.api.Product(organization=module_org).create() for _ in range(2)]
+    [target_sat.api.Repository(product=product).create() for product in products]
```

Comment:
It seems like this setup could be turned into a fixture with some parametrization, since it's very common throughout the file. Non-blocking.

Reply:
True. I expect this to happen on SyncPlans evaluation/refactor. Many tests may be merged together.